niworkflows/interfaces/surf.py

```diff
@@ -131,6 +132,15 @@ class GiftiNameSource(SimpleInterface):
     >>> res.outputs.out_name
     'space-fsaverage.R.func'
 
+    >>> namer = GiftiNameSource()
+    >>> namer.inputs.pattern = r'(?P<LR>[lr])h.(?P<space>\w+).gii'
```

Comment:
```suggestion
    >>> namer.inputs.pattern = r'(?P<hemi>[lr])h.(?P<space>\w+).gii'
```

Reply:
i think this was meant to adhere to the GIFTI standard, not BIDS

Reply:
not sure what you mean here? Does GIFTI define an "LR" metadata?

still, space and density are bids-like... certainly unrelated to GIFTI.

I would apply this change also for readability - LR could be easily confused with the pattern (lr) and the template (fsLR).